src/User32.Desktop/User32+SafeDCHandle.cs

```diff
@@ -26,8 +26,10 @@ public class SafeDCHandle : SafeHandle
             /// </summary>
             /// <param name="hWnd">The HWND this handle is associated with and must be released with.</param>
             /// <param name="hDC">The handle to the DC.</param>
-            public SafeDCHandle(IntPtr hWnd, IntPtr hDC)
-                : base(IntPtr.Zero, true)
+            /// <param name="ownsHandle"><see langword="true"/> to reliably release the handle during the finalization
+            /// phase; <see langword="false"/> to prevent reliable release.</param>
```

Comment:
I think I'd describe `false` as "to prevent this SafeHandle from releasing the native handle when it is disposed or finalized."
Saying it prevents "reliable release" sounds like it may or may not release it, rather than that it certainly will not.